docs/container.md

```diff
@@ -0,0 +1,313 @@
+---
+category: Develop
+previous: piwiks-ini-configuration
+next: plugin-settings
```

Comment:
I'm not sure where this would currently appear as `piwiks-ini-configuration` is under "Piwik In Depth" and `plugin-settings` under "Develop > Utils". 

Maybe we could have
* one guide `Dependency Injection` under "Develop > Utils" explaining how to get dependencies
* one guide `Configuration` under "Develop > Plugin Basics" describing the config section from the guide below and all the possibilities one has. This could be as well part of the `Dependency Injection` guide. I'm not sure if it's beneficial to have 2 separate guides for this or whether it is confusing. We can still link between them so one can find it in 2 places which might be better. Not sure!
* and the more detailed documentation part in "Piwik In Depth > Utils" next or in the configuration page?